spec/message_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'message'
+
+describe Message do
+  it "creates an instance of message" do
+    expect(subject).to be_instance_of Message
+  end
+end 
```

Comment:
you could think about changing the nature of the test to something like it "delivers a message with estimated delivery time" using expect(subject).to eq "thanks, order to be delivered before 17:00"